types/address/codec.go

```diff
@@ -0,0 +1,25 @@
+package address
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+type Codec interface {
+	// AddressStringToBytes decodes text to bytes
+	AddressStringToBytes(text string) ([]byte, error)
```

Comment:
```suggestion
	AddressStringToBytes(text string) ([]byte, error)
```

let's rename to `StringToBytes`

Reply:
Sure